File: src/codes/Users_Manual/controller_with_parallel.py
~# mypy: ignore-errors
~from pyautd3 import Controller, AUTD3, Static, Null
~from pyautd3.link.audit import Audit
~autd: Controller[Audit] = Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Audit.builder())
~m = Static()
~g = Null()
autd.send((m, g).with_parallel_threshold(0))

File: src/codes/Users_Manual/controller_with_parallel.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Driver.Datagram;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
~var m = new Static();
~var g = new Null();
autd.Send((m, g).WithParallelThreshold(0));